packages/google_maps_flutter/google_maps_flutter_android/test/google_maps_flutter_android_test.dart

```diff
@@ -774,6 +782,164 @@ void main() {
     },
   );
 
+  test('moveCamera calls through newCameraPosition', () async {
```

Comment:
`moveCamera calls though with expected newCameraPosition`; similarly with the methods below.